.github/workflows/package_pypi.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  push:
+  pull_request:
+  release:
+    types: [published]
+  schedule:
+    - cron: '0 1 * * *'
+
+name: PyPi packaging
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest ]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Install dependencies
+      uses: ./.github/actions/install-dependencies
+      if: runner.os == 'Linux'
+      with:
+        with-documentation: false
+        with-python-bindings: false
```

Comment:
Why is this false but for macOS it is true?

Reply:
For Linux, the compilation takes place within Docker and the software is thus installed in the Docker container. For macOS, compilation is done directly on the host.